pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,127 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
+
+    //storage-client
+    rpc PutChunks(ChunksData) returns (Nothing);
+    rpc GetChunks(Chunks) returns (ChunksResponse);
```

Comment:
This doesn't work yet. This requires that the upstream service has knowledge about the schema (to figure out which table to read from), but that shouldn't be the case. We should send the table info as well.

Reply:
Yeah fixed it. gRPC server will not be aware of `schemaConfig`. `Tablename `will be sent in `GetChunks `request field.